### ISSUE TITLE
Page 11 completely unclear to a novice which this books is targeting

### ISSUE THREAD

**Issue by michael-keyes** [karlseguin/the-little-go-book#77]
Page 11 completely unclear to a novice which this books is targeting

**Comment by diondresschers** [comment]
I don't think this book is written for a novice. Karl mentioned in the book itself to doubt to write this book, as there can be a big difference in the knowledge of the audience. Perhaps good to write in the beginning for who this book is intended.
